docs/src/main/sphinx/release/release-352.md

```diff
@@ -0,0 +1,88 @@
+* Improve performance of queries using {func}`rank()` window function. ({issue}`6333`)
+* Improve performance of {func}`sum` and {func}`avg` for `decimal` types. ({issue}`6951`)
+* Improve join performance. ({issue}`5981`)
+* Improve query planning time. ({issue}`6544`)
+* Fix window and streaming aggregation semantics regarding peer rows. Now peer rows are
+  not distinct from each other, while previously they had to be equal. ({issue}`6472`) 
+* Fix query failure when using element of `array(timestamp(p))` in a complex expression 
+  for `p` greater than 6. ({issue}`6350`)
+
+## Web UI
+
+* Support JWK with certificate chain for OAuth2 authorization. ({issue}`6428`)
+* Optionally verify JWT audience (`aud`) field for OAuth2 Web UI authentication. ({issue}`6501`)
+* Add scopes to `OAuth2` configuration. ({issue}`6580`)
+* Guard against replay attacks in `OAuth2` by using `nonce` cookie when `openid` scope is requested. ({issue}`6580`)
+
+## JDBC driver
+
+* Support user impersonation when using password-based authentication. This is configured 
+  using `sessionUser` connection URL parameter. Previously user impersonation was possible
+  only with other authentication mechanisms. ({issue}`6549`)
+* Add OAuth2 authentication support. ({issue}`6576`)
+
+## Docker image
+
+* Remove support for configuration directory `/usr/lib/trino/etc`. The configuration should be provided in `/etc/trino`.
+
+## CLI
+
+* Support user impersonation when using password-based authentication. This is configured
+  using `--session-user` command line parameter. Previously user impersonation was possible 
+  only with other authentication mechanisms. ({issue}`6567`)
+
+## BigQuery connector
+
+* Fix query failure when executing `count(*)` without `WHERE` clause for views. ({issue}`6706`)
+
+## Hive connector
+
+* Support reading timestamp with microsecond or nanosecond precision (as configured with 
+  `hive.timestamp-precision` property) nested within `array`, `map` or `struct` data types. ({issue}`5195`)
+* Support reading from table in Sequencefile format that uses LZO compression. ({issue}`6452`)
+* Expose AWS HTTP Client stats via JMX. ({issue}`6503`)
+
+## Iceberg connector
+
+* Optimize cardinality-insensitive aggregations (max, min, distinct, approx_distinct) over 
+  identity partition columns with `optimizer.optimize-metadata-queries` config property 
+  or `optimize_metadata_queries` session property. ({issue}`5199`)
+* Provide  `use_file_size_from_metadata` catalog session property and `iceberg.use-file-size-from-metadata` 
+  config property to fix query failures on tables with wrong file sizes stored in the metadata. ({issue}`6369`)
+ 
+## Kinesis connector
+
+* Support GZIP message compression. ({issue}`6442`)
+
+## MySQL connector
+
+* Improve performance for certain complex queries involving aggregation and predicates (e.g. `HAVING` clause)
+  by pushing the aggregation and predicates computation into the remote database. ({issue}`6667`)
```

Comment:
@findepi how about mentioning the actual name instead of "the remote database" in each section?

Reply:
discussed offline, current wording also seems fine to go with.